src/error/mod.rs

```diff
@@ -72,7 +72,7 @@ impl Display for RoverError {
                 suggestion_descriptor_message.push(' ');
             }
             let suggestion_descriptor = Cyan.bold().paint(&suggestion_descriptor_message);
-            write!(formatter, "{} {}", suggestion_descriptor, suggestion)?;
+            writeln!(formatter, "{} {}", suggestion_descriptor, suggestion)?;
```

Comment:
This was mentioned by Jesse- 

zsh prints a `%` after a command if it doesn't end with a newline